### ISSUE TITLE
About the width of the model

### ISSUE THREAD

**Issue by wsy-yjys** [huawei-noah/VanillaNet#8]
Your work is exciting! but I have a question about the width of the model，Have you tried to reduce the width of the model, why is the width of the  smallest  Vanillanet_5 need [128*4, 256*4, 512*4, 1024*4]? thank you~

**Comment by HantingChen** [comment]
Thank you for your question! Our aim is to maximize the parallelism of the network and reduce the number of layers as much as possible to achieve better speed on the GPU. The current number of channels represents a good trade-off. Reducing the channels may cause a significant decrease in accuracy with relatively little gain in speed, hence our choice of these specific channel numbers. Of course, you're welcome to try different channel numbers and train the network yourself to explore further.

**Comment by HantingChen** [comment]
Thank you for your question! Our aim is to maximize the parallelism of the network and reduce the number of layers as much as possible to achieve better speed on the GPU. The current number of channels represents a good trade-off. Reducing the channels may cause a significant decrease in accuracy with relatively little gain in speed, hence our choice of these specific channel numbers. Of course, you're welcome to try different channel numbers and train the network yourself to explore further.

**Comment by wsy-yjys** [comment]
Thank you for your reply!  as you say “maximize the parallelism of the network”, so if the optimal channels of layer in model is related to the hardware performance, whether the optimal number of channels is different on different hardware? Especially the edge end Gpus？

**Comment by HantingChen** [comment]
Absolutely, the optimal number of channels is indeed influenced by the specific hardware in use. Particularly for edge-end GPUs, which may have less parallel processing capability compared to more robust systems, a smaller number of channels could be more suitable. It's always crucial to take into account the specific hardware characteristics when seeking to optimize for the best performance.

**Comment by wsy-yjys** [comment]
I see. Thank you very much for your reply~
